packages/docs/middleware/redirects.global.ts

```diff
@@ -0,0 +1,15 @@
+// Is used to test following paths /en, /en/, /en/something, /ru, /ru/, /ru/something
+const langTestRegex = /^\/(en|ru)\/?$|^\/(en|ru)\/.*/i
```

Comment:
```suggestion
const langTestRegex = /^\/(en|ru)/i
```

If this regex is used in `.test` it checks if string starts with `/en` or `/ru`. Removed part doesn't make sense to me.

You can use `/^\/(en|ru)\/?/i` for both check if string starts with prefix and to remove this prefix.

Reply:
It doens't work for the case when you have something like `/rumba` or `/end`. It will be replaced with `/mba` and `/d`

Reply:
There is an exact check for `/en` or `/en/` OR for `/en/something`